ui/imports/shared/status/StatusSticker.qml

```diff
@@ -9,15 +9,16 @@ Loader {
     property string stickerData: ""
     property int imageHeight: 140
     property int imageWidth: 140
-    signal loaded()
+
+    signal stickerLoaded()
 
     id: root
     active: contentType === Constants.messageContentType.stickerType
 
     sourceComponent: Component {
         ImageLoader {
             color: root.color
-            onLoaded: root.loaded()
+            onLoaded: root.stickerLoaded()
```

Comment:
Same problem... as with the other "StatusSticker"; yeah, we have 2 components called "StatusSticker" :cry: